README.md

```diff
@@ -1 +1,195 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Project Name
+
+#### To Do List Project
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML, CSS, and JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- Project with webpack.
+- Used HTML Webpack Plugin to generate HTML files.
+- Array of leaders is used to populate the HTML.
+- Function to iterate over tasks to populate HTML according to task index order.
+- Dynamically generate leader list.
+- Styling used following template.
+- Code is created in src folder and dist folder is generated automatically.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+To clone the repository in local environment try following steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+### Setup
+
+Clone the repository using [this link](https://github.com/PrangonGhose/To-do-list.git)
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/PrangonGhose/Leaderboard.git
+```
+### Get into development
+
+In the terminal, run these commands to get into development.
+```
+$ cd To-do-list
+
+$ npm init -y
+
+$ npm install
+```
+### Usage
+
+This website is applicable for both mobile and desktop version.
+
+### Run tests
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Prangon Ghose**
+
+- GitHub: [@PrangonGhose](https://github.com/PrangonGhose)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Styling of the webpage**
+- [ ] **Animation to add functionality**
+- [ ] **Adding responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! Add suggestions by opening new issues.
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] **[OPTIONAL]:** I suggest you add a link to the issues page of this repo, so people that want to contribute can have an easy way to create new issues! 🤓 🛠️